source/GameInterface.Tests/Bootstrap/Patches/DefaultMapWeatherModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using TaleWorlds.CampaignSystem.ComponentInterfaces;
+using TaleWorlds.CampaignSystem.GameComponents;
+using static TaleWorlds.CampaignSystem.ComponentInterfaces.MapWeatherModel;
+
+namespace GameInterface.Tests.Bootstrap.Patches;
+
+[HarmonyPatch(typeof(DefaultMapWeatherModel))]
+internal class DefaultMapWeatherModelPatches
+{
+
+    [HarmonyPatch(nameof(DefaultMapWeatherModel.GetWeatherEventInPosition))]
+    [HarmonyPrefix]
+    static bool GetWeatherEventInPositionPrefix(ref WeatherEvent __result)
+    {
+        Array values = Enum.GetValues(typeof(WeatherEvent));
+        Random random = new Random();
```

Comment:
Is there a reason to use Random here? Wouldn't this create different weather types for every player?

Reply:
This is for testing only, these patches are located in GameInterface.Tests

Reply:
Haha oppps, missed that.